<ui version="4.0" >
 <class>ScheduleSmSManage</class>
 <widget class="QMainWindow" name="ScheduleSmSManage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="DeleteSCM" >
    <property name="geometry" >
     <rect>
      <x>310</x>
      <y>10</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>delete</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="view" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>40</y>
      <width>401</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text" >
      <string>选中</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>收件人</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>发送时间</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>短信内容</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="CreateSCM" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChooseAllSCM" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>10</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>choose all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshSCM" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>10</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>refresh</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
